lib/commons/dom/focus-disabled.js

```diff
@@ -14,6 +14,24 @@ function focusDisabled(el) {
     return true;
   }
 
+  let parentNode = vNode.parent;
+  while (parentNode) {
+    // if a form element is in a legend, that element will not be disabled even if the fieldset is
+    // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset
+    if (parentNode.props.nodeName === 'legend') {
+      parentNode = null;
+      continue;
```

Comment:
This could just be a `break` rather than setting `parentNode` to null to quit the loop in the next iteration and then continuing.

```suggestion
      break;
```